api/v4/source/definitions.yaml

```diff
@@ -3637,7 +3637,7 @@ components:
         state:
           description: The current state of the installation
           type: string
-    UserLimits:
+    AppLimits:
```

Comment:
Could this be `ServerLimits` instead? Thinking about the possible confusion with Apps framework for plugins that @M-ZubairAhmed called out here https://hub.mattermost.com/private-core/pl/a3jz8m9ne3n7px3pz3xhc91kco

Reply:
Done!